### DISCUSSION TITLE
How to enable EXTRA

### DISCUSSION THREAD

**Discussion by smason-apex** [manna-harbour/miryoku#146] [Q&A] [answered]
Apologies for what may be a dumb question but how exactly do I enable the EXTRA layer (or any of the other Additional Features)? 

I'm on default settings (for the most part). With my left thumb, I press the primary thumb key and then while holding it I press the top key in the middle column with my left hand. But it's still coming up colemak after I release it. Every other layer works perfectly fine.

The only change I made to my config was to enable `MIRYOKU_KLUDGE_THUMBCOMBOS` (I'm on a ferris sweep) and the mac clipboard.

**Comment by smason-apex** [comment]
My mistake. I needed to build from miryoku main and not qmk main.

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#branches for a description of the branches. Some changes will be made to Miryoku QMK soon, followed by a PR. After that those features will be available in QMK master. But in general, the documentation at https://github.com/manna-harbour/miryoku describes the version of Miryoku QMK at https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku.

`MIRYOKU_KLUDGE_THUMBCOMBOS` should be enabled automatically for ferris. See https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#ferris and https://github.com/manna-harbour/miryoku_qmk/blob/miryoku/keyboards/ferris/keymaps/manna-harbour_miryoku/rules.mk.
